.github/workflows/verify_dco.yaml

```diff
@@ -0,0 +1,60 @@
+name: Verify DCO
+on:
+  schedule:
+    - cron:  '*/15 * * * *'
+
+jobs:
+  dco-helper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Verify DCO status for open pull requests
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const owner = "backstage";
+            const repo = "backstage";
+            const pulls = await github.paginate(github.rest.pulls.list, {
+              state: "open",
+              owner,
+              repo,
+            });
+
+            for (const pull of pulls) {
+              // Pick out the PRs that have the DCO check
+              const checks = await github.rest.checks.listForRef({
+                owner,
+                repo,
+                ref: pull.head.sha,
+                check_name: "DCO",
+                status: "completed",
+              });
+              // Skip if there are no checks
+              if (!checks.data.check_runs.length) {
+                continue;
+              }
+              // Skip if the conclusion is not action_required
+              if (checks.data.check_runs[0].conclusion !== "action_required") {
+                console.log(`No checks found for PR #${pull.number}, skipping`);
+                continue;
+              }
+
+              const comments = await github.paginate(github.rest.issues.listComments, {
+                owner,
+                repo,
+                issue_number: pull.number,
+              });
+
+              if (comments.find((c) => c.user.login === "github-actions[bot]")) {
```

Comment:
if there's ANY comment by this bot? 🤔

Reply:
yeah, felt safest to do for now to avoid adding a new comment every time 😅

Reply:
Hmm I think all the `stale-bot` messages come from that bot too, so it might not work on every current PR that we have open if it's been staled? Is it expensive to get the text contents of each message and see if it smells like a DCO link from this bot?

Reply:
I've added a `<dco-helper>` tag that's not visible by the user